ansible_runner/runner.py

```diff
@@ -186,6 +229,10 @@ def run(self):
                 f.write(str(data))
         if self.config.directory_isolation_path and self.config.directory_isolation_cleanup:
             shutil.rmtree(self.config.directory_isolation_path)
+        if self.config.resource_profiling:
+            cmd = ['cgdelete', '-g', 'cpuacct,memory,pids:ansible-runner-{}'.format(self.config.ident)]
+            proc = subprocess.Popen(cmd, shell=True)
```

Comment:
If you don't actually need stdout or care how this exited, using `subprocess.call` as above is simpler.